documentation/kyverno-cli.md

```diff
@@ -40,39 +39,58 @@ yay -S kyverno-git
 
 Prints the version of kyverno used by the CLI.
 
-Example: 
+Example:
+
 ```
 kyverno version
 ```
 
 #### Validate
-Validates a policy, can validate multiple policy resource description files or even an entire folder containing policy resource description 
+
+Validates a policy, can validate multiple policy resource description files or even an entire folder containing policy resource description
 files. Currently supports files with resource description in yaml.
```

Comment:
please change yaml to YAML everywhere in the doc

Reply:
I have changed it.